storage/store.go

```diff
@@ -1013,7 +1012,7 @@ func (s *Store) maybeResolveWriteIntentError(rng *Range, method string, args pro
 			Txn:          args.Header().Txn,
 		},
 		PusheeTxn: wiErr.Txn,
-		Abort:     proto.IsReadWrite(method), // abort if cmd is read/write
+		Abort:     proto.IsWrite(args), // abort if cmd is read/write
```

Comment:
Note that the comment disagrees with the code, but the behavior here is unchanged as `proto.IsReadWrite(method)` was only checking to see if `method` was in the `WriteMethods` set. What is correct? The comment or the code?